lib/resources/r4/related_person_patch.yaml

```diff
@@ -131,6 +140,44 @@ operations:
         <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
       </ul>
 
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship
+    description: Replace the `period` and `relation` extensions of the relationship at the given <code>{index}</code>.
+    example: |
+      {
+        "op": "replace",
+        "path": "/relationship/0/extension",
+        "value": [
+          {
+            "valuePeriod": {
+              "start": "2016-12-30T22:26:08.000Z",
+              "end": "2036-12-30T22:26:08.000Z"
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
+          },
+          {
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40",
+                  "code": "155",
+                }
+              ]
+            },
```

Comment:
use the standard codes example here instead of proprietary code.
cc: @alexbeckner1

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/062a7f28bf665d671d55b5d50c1f8fa7067a2349#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR167-R168